CWE/Common/Embeds.cs

```diff
@@ -132,77 +133,140 @@ public static Embed GetRequestEmbed(Request request)
                 .AddField("Member", $"<@{campaign.User}>", true)
                 .AddField("Type", campaign.Type, true)
                 .AddField("Reason", campaign.Reason)
-                .AddField("Voting", reason ?? $"This campaign was denied because it didn't receive enough votes within 24 hours.")
+                .AddField("Voting",
+                    reason ?? "This campaign was denied because it didn't receive enough votes within 24 hours.")
                 .WithColor(Colors.Error)
                 .Build();
             return embed;
         }
+
+        public static Embed HugAsync(string title, string description)
+        {
+            var embed = new EmbedBuilder()
+                .WithColor(Colors.Hug)
+                .WithDescription(description)
+                .WithTitle(title)
+                .WithThumbnailUrl(
+                    Icons.Hug)
+                .Build();
+            return embed;
+        }
+
+        public static Embed eightBallEmbed(string question, string eightResponse, SocketUser user)
+        {
+            var embed = new EmbedBuilder()
+                .WithColor(Colors.eightBall)
+                .WithTitle("Magic 8-Ball")
+                .WithDescription(
+                    $"{user.Mention}'s Question For The Magic 8-Ball: `{question}`\n\nThe Magic 8-Ball's Response Is: `{eightResponse}\n`")
+                .WithFooter($"8-Ball response for: {user.Username + "#" + user.Discriminator}")
+                .WithThumbnailUrl(
+                    Icons.eightBall)
+                .WithCurrentTimestamp()
+                .Build();
+            return embed;
+        }
+
+        public static Embed CoinflipEmbed(string output, string user)
+        {
+            var embed = new EmbedBuilder()
+                .WithColor(Colors.Coinflip)
+                .WithTitle("Coinflip")
+                .WithDescription($"The coin landed on __**{output}**__")
+                .WithThumbnailUrl(Icons.Coinflip)
+                .WithCurrentTimestamp()
+                .WithFooter($"Coin flipped by {user}")
+                .Build();
+            return embed;
+        }
+
+        public static Embed pollEmbed(string description, SocketUser user)
+        {
+            var embed = new EmbedBuilder()
+                .WithAuthor(x =>
+                {
+                    x
+                        .WithIconUrl(user.GetAvatarUrl())
+                        .WithName(user.Username + "#" + user.Discriminator);
+                })
+                .WithTitle("__**Poll**__")
+                .WithColor(Colors.Poll)
+                .WithDescription($"**{description}**")
+                .WithThumbnailUrl(
+                    Icons.Poll)
+                .WithCurrentTimestamp()
+                .WithFooter($"Poll started by {user.Username + "#" + user.Discriminator}")
+                .Build();
+            return embed;
+        }
     }
-}
+}
```

Comment:
File again contains way too many changes, as again these embeds are only used in their respective commands, there is no need to add them to this class.